shared/profile/index.native.js

```diff
@@ -82,7 +82,7 @@ const ShowcasedTeamRow = ({
       <Text style={{color: globalColors.black_75}} type="BodySemiboldLink">
         {team.fqName}
       </Text>
-      {team.open && <Meta style={styleMeta} title="OPEN" />}
+      {team.open && <Meta style={styleMeta} backgroundColor={globalColors.green} title="open" />}
```

Comment:
i changed title to lowercase so its more clear that component itself uppercases this stuff